app/src/main/java/com/imashnake/animite/features/MainActivity.kt

```diff
@@ -39,65 +44,72 @@ class MainActivity : ComponentActivity() {
         WindowCompat.setDecorFitsSystemWindows(window, false)
 
         setContent {
-            val systemUiController = rememberSystemUiController()
-            SideEffect {
-                systemUiController.setSystemBarsColor(Color.Transparent, darkIcons = false)
-            }
+            AnimiteTheme {
+                val systemUiController = rememberSystemUiController()
+                SideEffect {
+                    systemUiController.setSystemBarsColor(Color.Transparent, darkIcons = false)
```

Comment:
Is this something that `isSystemInDarkTheme()` can handle?

Reply:
Light mode isn't supported yet.